sql/moz-fx-data-shared-prod/telemetry_dev_cycle_external/expirements_stats_v1/query.py

```diff
@@ -0,0 +1,153 @@
+"""Experiment data downloaded from APIs, clean and upload to BigQuery."""
+
+import datetime
+import logging
+
+import click
+import requests
+from google.cloud import bigquery
+
+API_BASE_URL_EXPERIMENTS = "https://experimenter.services.mozilla.com"
+API_BASE_URL_METRIC_HUB = "https://github.com/mozilla/metric-hub/tree/main/jetstream"
+SCHEMA = [
```

Comment:
Could we do something like this instead here?
https://github.com/mozilla/bigquery-etl/blob/869aad3b2d21492bc95585d3b5596c45dd8e9910/sql/moz-fx-data-shared-prod/telemetry_derived/crash_signatures_v1/query.py#L9-L12

This could allow us to have a `schema.yaml` which then could be used in the staging deploy when running checks?